src/extension/src/azure/azure-app-service/appServiceProvider.ts

```diff
@@ -46,10 +46,11 @@ const APP_SERVICE_DEPLOYMENT_SUFFIX = "-app-service";
 export class AppServiceProvider {
   private webClient: WebSiteManagementClient | undefined;
 
+  // returns the id of the instance created
   public async createWebApp(
     selections: AppServiceSelections,
     appPath: string
-  ): Promise<void> {
+  ): Promise<string | undefined> {
```

Comment:
Why is it string or undefined?

Reply:
the result.id is of type string or undefined